### PR TITLE
Feature/lightweight

### PR DESCRIPTION
**Pull request by boubaker**

### CODE REVIEW

File: exo.jcr.framework.web/src/main/java/org/exoplatform/frameworks/jcr/web/InstallJCRServletContextListener.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+import javax.servlet.*;
+import javax.servlet.FilterRegistration.Dynamic;
+
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+
+/**
+ * This is a service that is injected via Kernel Container to install a JCR Web
+ * Filter
```
[the remark this answers is not in the document]
Comment (boubaker) [reply exo.jcr.framework.web/src/main/java/org/exoplatform/frameworks/jcr/web/InstallJCRServletContextListener.java]: done

[the remark this answers is not in the document]
Comment (thomasdelhomenie) [reply exo.jcr.framework.web/src/main/java/org/exoplatform/frameworks/jcr/web/InstallJCRServletContextListener.java]: "it must update the webapp web.xml file directly" should be "it must not update the webapp web.xml file directly" no ?

[the remark this answers is not in the document]
Comment (boubaker) [reply exo.jcr.framework.web/src/main/java/org/exoplatform/frameworks/jcr/web/InstallJCRServletContextListener.java]: done

[the remark this answers is not in the document]
Comment (thomasdelhomenie) [reply exo.jcr.framework.web/src/main/java/org/exoplatform/frameworks/jcr/web/InstallJCRServletContextListener.java]: :+1:

**Pull request merged**